app/models/ext_management_system.rb

```diff
@@ -694,6 +694,10 @@ def conversion_hosts
     host_conversion_hosts + vm_conversion_hosts
   end
 
+  def total_active_tasks
+    host_conversion_hosts.sum(:total_active_tasks) + vm_conversion_hosts.sum(:total_active_tasks)
+  end
+
```

Comment:
Since this is on the EMS model, and since other models could potentially need their own list of active tasks, perhaps it would be best to rename this to `total_active_conversion_host_tasks`.